src/changelog.ts

```diff
@@ -13,7 +13,7 @@ const changelogDescription = `All notable changes to this project will be docume
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).`;
 
-interface ReleaseMetadata {
+type ReleaseMetadata = {
```

Comment:
I've seen this change made in a few commits and I'm curious about the motivation behind the shift from interface to type. Is this something we are slowly changing across the project, or is there a specific need that we are addressing?

Reply:
See here for more context: https://github.com/MetaMask/eslint-config/pull/216 (and [this Slack thread](https://consensys.slack.com/archives/G1L7H42BT/p1638221772259900) for even more context)

Primarily it was for consistency. No specific need being addressed. Erik referenced some difficulties encountered working with interfaces and our `Json` type, but I can't recall the details on that.

Reply:
This is great, thanks for the context @Gudahtt!

Reply:
> Erik referenced some difficulties encountered working with interfaces and our `Json` type, but I can't recall the details on that.

It has to do with interfaces not having an index signature, IIRC.